services/server/src/sourcify-chains-default.json

```diff
@@ -2252,5 +2252,37 @@
   "26100": {
     "sourcifyName": "Quantum Portal Network",
     "supported": true
+  },
+  "7200": {
+    "sourcifyName": "exSat Mainnet",  
+    "supported": true,  
+    "fetchContractCreationTxUsing": {
+      "etherscanApi": false,  
+      "blockscoutApi": {
+        "url": "https://scan.exsat.network/"
+      },
+      "blockscoutScrape": {
+        "url": "https://scan.exsat.network/"
+      }
+    },
+    "rpc": [
+      "https://evm.exsat.network"
+    ]
```

Comment:
Please remove these fields. There's already the blockscoutApi so no need to scrape. The rpc will be fetched from the `chains.json` file we periodically sync from https://chainid.network/chains

Reply:
Also valid for the other chain below

Reply:
> Please remove these fields. There's already the blockscoutApi so no need to scrape. The rpc will be fetched from the `chains.json` file we periodically sync from https://chainid.network/chains

Removed, please review, thanks.

Reply:
You can add the `blockscoutApi` field again. `blockscoutScrape` should just not be there when we already have `blockscoutApi`.

Reply:
> You can add the `blockscoutApi` field again. `blockscoutScrape` should just not be there when we already have `blockscoutApi`.

Thank you for reminding me, I just added it again, please check again.